mindsdb/helm/mindsdb/templates/deployment.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Isn't this a stateful application?  Deployments are usually not the right construct here, and should use a statefulset to simplify volume provisioning/binding.   Also it might be useful to search for and use an existing mindsdb helm chart instead of writing it fresh if you're a bit unfamiliar with k8s.

Reply:
> Isn't this a stateful application?  Deployments are usually not the right construct here, and should use a statefulset to simplify volume provisioning/binding.   Also it might be useful to search for and use an existing mindsdb helm chart instead of writing it fresh if you're a bit unfamiliar with k8s.

This is an existing mindsdb helm chart. It's on their repo, although they say it was an open source contribution from someone.

Reply:
Does the server store any data underneath or is it actually effectively stateless?

Reply:
it does

> Does the server store any data underneath or is it actually effectively stateless?

it's stateful, i'll cange it